.trunk/trunk.yaml

```diff
@@ -0,0 +1,43 @@
+version: 0.1
+cli:
+  version: 0.1.31-alpha
+  sha256:
+    darwin_x86_64: 15f297b0657b615609adfd1a30b99a6210a6ff7c52dc8bdfeb0053a8aff7a9bc
```

Comment:
Most of our developers use M1 Macbooks at this point.

Reply:
As do we. Love the M1. Most analyzers haven't been recompiled for arm mac, but we plan on compiling a native arm build for trunk in the future.